api/namex/resources/name_requests/name_requests.py

```diff
@@ -145,6 +145,22 @@ def post(self):
             self.initialize()
             nr_svc = self.nr_service
 
+            name_search_string = ""
+            user_email = ""
+            # user id 
+            submitter=nr_svc.request_data.get("applicants")
```

Comment:
coding format is part of code review. could you please add space before and after "="